Classes/Projection/Doctrine/AbstractDoctrineProjector.php

```diff
@@ -105,18 +122,64 @@ protected function remove($object)
      * @return void
      * @api
      */
-   public function reset()
-   {
-       $this->projectionPersistenceManager->drop($this->getReadModelClassName());
-   }
+    public function reset()
+    {
+        $this->projectionPersistenceManager->drop($this->getReadModelClassName());
+    }
 
-   /**
-    * If this projection is currently empty
-    *
-    * @return bool
-    */
-   public function isEmpty()
-   {
-       return $this->projectionPersistenceManager->count($this->getReadModelClassName()) === 0;
-   }
+    /**
+     * If this projection is currently empty
+     *
+     * @return bool
+     */
+    public function isEmpty()
+    {
+        return $this->projectionPersistenceManager->count($this->getReadModelClassName()) === 0;
+    }
+
+    /**
+     * Returns the last seen sequence number of events which has been applied to the concrete event listener.
+     *
+     * @return int
+     */
+    public function getHighestAppliedSequenceNumber(): int
+    {
+        $projectorIdentifier = $this->renderProjectorIdentifier(get_class($this));
```

Comment:
Wouldn't it make sense to use the actual *projection* identifier? (pass it in from the `EventPublisher`)